test/kwtool_web/controllers/api/v1/upload_controller_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule KwtoolWeb.Api.V1.UploadControllerTest do
+  use KwtoolWeb.ConnCase, async: true
+
+  import KwtoolWeb.Support.ConnHelper
+
+  describe "post create/2" do
```

Comment:
Do we have the test for unauthenticated requests? 🤔 💭

Reply:
yes, i added here https://github.com/byhbt/kwtool/pull/46/commits/2e8acaeedc3ea29ce12702428cc668e82d327bad